docs/styling-engines.md

```diff
@@ -0,0 +1,25 @@
+# 🕴️ Styling engines
+
+### Using UIToolkit in external windows 🪟
+UIToolkit's `Dropdown` and `Tooltip` components have libraries that inject styling with `@emotion`. If you try using these components in a a window outside the main one (popout for example) the styling will only be applied to the main document. So the components needs to be wrapped in a CacheProvider with the container argument set to the document where the styling will be implemented. For example:
```

Comment:
Does it work without a `CacheProvider` in the main window? Isn't that only in the special case where the user happens to be an extension deployed in SFE-Lite, where SFE-Lite already added it?
So in fact, every user of UI Toolkit needs to add it?

Reply:
Yes, it should work without a `CacheProvider` in the main window cause that's where the styling will be injected by default. But anyone using UIToolkit in external windows should wrap that usage in a `CacheProvider` with the new injectionPoint.